cypress/integration/E2E_tests/MessagingTests/messaging.spec.js

```diff
@@ -0,0 +1,29 @@
+describe("On the messages page, can", () => {
+  before(() => {
+    cy.login();
+    cy.wait(2000);
+    cy.fixture("url").then((jsonFile) => {
+      cy.visit(
+        `${jsonFile.HOMEPAGE}:${process.env.PORT || 8081}${jsonFile.MESSAGES}`
+      );
+    });
+  });
+  it("display messages", () => {
+    cy.get("[data-cy=messages-window]").should("be.visible");
+  });
+  it("search for people", () => {
+    cy.get("[data-cy=search]").click().type("Jane Doe");
+  });
+  it("create new chat", () => {
+    cy.get("[data-cy=new-message]").click();
+  });
```

Comment:
I don't see an assertion of some sort in these tests. Is it possible to add one? Or is the feature under test incomplete? Something like after searching "Jane Doe" See if jane is actually displayed, and after clicking new-message something is visible.

Reply:
Yeah the only reason is because right now it's essentially only the design that's implemented, with no functionality. Sending messages doesn't work, selecting a specific conversation doesn't work, creating a new convo doesn't work and search is currently just a text-box that has an autocomplete but isn't linked to searching functionality. I can still assert that the text in the search box is present in the search box though

Reply:
All good then!